task/minio-cli/0.1/minio-cli.yaml

```diff
@@ -0,0 +1,49 @@
+---
+apiVersion: tekton.dev/v1beta1
+kind: Task
+metadata:
+  name: minio-cli
+  labels:
+    app.kubernetes.io/version: "0.1"
+  annotations:
+    tekton.dev/pipelines.minVersion: "0.12.1"
+    tekton.dev/tags: cli
+    tekton.dev/displayName: "minio cli"
+spec:
+  description: >-
+   MinIO is a High Performance Object Storage released under Apache License v2.0.
+   It is API compatible with Amazon S3 cloud storage service.
+
+   This task lists all the buckets available on the MinIO server.
+
+  params:
+  - name: server-url
+    type: string
+    description: MinIO server URL
+  - name: server-alias
+    type: string
+    description: MinIO server alias
+  - name: key-tokens
+    type: string
+    description: Secret for the MinIO server
+  - name: cli-command
+    description: The minio cli command to tun
+    type: string
+    default: "--help"
```

Comment:
shouldn't this be an `array` so that we can pass multiple arguments example `--json ls` or probably you can keep this but try adding an example so that users can come to know how to pass multiple arguments via this `param`